package.json

```diff
@@ -48,11 +48,11 @@
     "docker-toolbelt": "^3.3.7",
     "dockerode": "^2.5.8",
     "lodash": "^4.17.4",
-    "resin-bundle-resolve": "^4.0.0",
```

Comment:
Pending review of `resin-bundle-resolve` PR:
https://github.com/balena-io-modules/resin-bundle-resolve/pull/36